mungegithub/mungers/release-note-label.go

```diff
@@ -34,16 +34,17 @@ const (
 	releaseNote               = "release-note"
 	releaseNoteNone           = "release-note-none"
 	releaseNoteActionRequired = "release-note-action-required"
+	releaseNoteExperimental   = "release-note-experimental"
 
 	releaseNoteFormat = `Removing LGTM because the release note process has not been followed.
-One of the following labels is required %q, %q, or %q
-Please see: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes`
-	parentReleaseNoteFormat = `The 'parent' PR of cherry-picks must have either the %q or %q label or this PR must follow the release note process.`
+One of the following labels is required %q, %q, %q or %q.
+Please see: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes.`
+	parentReleaseNoteFormat = `The 'parent' PR of a cherry-pick PR must have one of the %q, %q or %q labels or this PR must then follow the standard/parent release note labeling requirement.`
 )
 
 var (
-	releaseNoteBody       = fmt.Sprintf(releaseNoteFormat, releaseNote, releaseNoteNone, releaseNoteActionRequired)
-	parentReleaseNoteBody = fmt.Sprintf(parentReleaseNoteFormat, releaseNote, releaseNoteActionRequired)
+	releaseNoteBody       = fmt.Sprintf(releaseNoteFormat, releaseNote, releaseNoteActionRequired, releaseNoteExperimental, releaseNoteNone)
+	parentReleaseNoteBody = fmt.Sprintf(parentReleaseNoteFormat, releaseNote, releaseNoteActionRequired, releaseNoteExperimental)
```

Comment:
"experimental" is just one of the available release note types we want to cover per #882.  I suspect as a general rule we don't want to include "experimental" things on a release branch, but in practice there will be times we want them and want to leave that judgement up to the PR owner and reviewer.  Thoughts @bgrant0607 ?